sdk/build.gradle

```diff
@@ -6,13 +6,13 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    compile 'org.jetbrains.kotlin:kotlin-reflect:1.3.21'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
-    compile 'com.fasterxml.jackson.module:jackson-module-kotlin:2.9.8'
-    compile 'com.fasterxml.jackson.module:jackson-module-parameter-names:2.9.8'
+    implementation 'org.jetbrains.kotlin:kotlin-reflect:1.3.21'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
+    implementation 'com.fasterxml.jackson.module:jackson-module-kotlin:2.9.8'
+    implementation 'com.fasterxml.jackson.module:jackson-module-parameter-names:2.9.8'
     testImplementation 'org.jetbrains.kotlin:kotlin-test-junit:1.3.11'
     testImplementation 'junit:junit:4.12'
-    test 'org.assertj:assertj-core:3.9.1'
+    testImplementation 'org.assertj:assertj-core:3.9.1'
```

Comment:
test is not supported by gradle any more. I would suggest using the bundled gradle-wrapper to build this project.
https://docs.gradle.org/current/userguide/java_library_plugin.html